php-frontend/src/main/java/org/sonar/plugins/php/api/tree/expression/FunctionExpressionTree.java

```diff
@@ -39,7 +48,7 @@
   @Override @Nullable
   SyntaxToken referenceToken();
 
-  ParameterListTree parameters();
+  ArgumentListTree parameters();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)